### PR TITLE
cotton: add page

### PR DESCRIPTION
**Pull request by koungkub**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/cotton.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# cotton
+
+> Markdown test specification runner.
+> More information: https://github.com/chonla/cotton.
```
Comment (sbrl) [inline pages/common/cotton.md]: Small formatting correction.
```suggestion
> More information: <https://github.com/chonla/cotton>.
```

File: pages/common/cotton.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Specific base url:
+
+`cotton -u {{base_url}} {{file}}.md`
+
+- Disable certificate verification (insecure mode):
+
+`cotton -u {{base_url}} -i {{file}}.md`
+
+- Stop running when failed:
```
Comment (sbrl) [inline pages/common/cotton.md]: Perhaps this will read better?
```suggestion
- Stop running when a test fails:
```

File: pages/common/cotton.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# cotton
+
+> Markdown test specification runner.
+> More information: https://github.com/chonla/cotton.
+
+- Specific base url:
```
Comment (sbrl) [inline pages/common/cotton.md]: Perhaps this will read better?
```suggestion
- Use a specific base url:
```

**Comment by koungkub** [comment]
Thanks you too @sbrl

**Pull request merged**